homeassistant/components/androidtv_remote/__init__.py

```diff
@@ -39,7 +39,7 @@ def is_available_updated(is_available: bool) -> None:
                 "Reconnected to %s at %s", entry.data[CONF_NAME], entry.data[CONF_HOST]
             )
         else:
-            _LOGGER.warning(
+            _LOGGER.debug(
```

Comment:
I had to add this because the guideline says: Handles device/service unavailable. Log a warning once when unavailable, log once when reconnected.

Reply:
The keyword here is *once* and it should probably be emphasized more :-) Without looking deeper into the code, this probably will probably log even when the device is already marked as unavailable in homeassistant?

Reply:
Yes this is supposed to be logged once after the device disconnects. If the device manages to connect and a few seconds later it disconnects again this will be logged again.

Reply:
And if the device stays disconnected this won't be logged again per the guideline.